client/index.html

```diff
@@ -356,25 +357,352 @@ <h1 class="title">Settings</h1>
+							<div class="subject">
+								<code>/nick newnick</code>
+							</div>
+							<div class="description">
+								<p>Change your nickname on the current network.</p>
+							</div>
+						</div>
+
+						<div class="help-item">
+							<div class="subject">
+								<code>/notice channel message</code>
+							</div>
+							<div class="description">
+								<p>Sends a notice message to the specified channel.</p>
+							</div>
+						</div>
+
+						<div class="help-item">
+							<div class="subject">
+								<code>/op nick [...nick]</code>
+							</div>
+							<div class="description">
+								<p>
+									Give op (<code>+o</code>) to one or several users in the
+									current channel.
+								</p>
+							</div>
+						</div>
+
+						<div class="help-item">
+							<div class="subject">
+								<code>/part</code>
+							</div>
+							<div class="description">
+								<p>Close the current channel or private message window.</p>
+								<p>Aliases: <code>/close</code>, <code>/leave</code></p>
+							</div>
+						</div>
+
+						<div class="help-item">
+							<div class="subject">
+								<code>/query nick</code>
+							</div>
+							<div class="description">
+								<p>Send a private message to the specified user.</p>
+							</div>
+						</div>
+
+						<div class="help-item">
+							<div class="subject">
+								<code>/quit [message]</code>
+							</div>
+							<div class="description">
+								<p>
+									Disconnect from the current network with an optional message.
+								</p>
+							</div>
+						</div>
+
+						<div class="help-item">
+							<div class="subject">
+								<code>/raw message</code>
+							</div>
+							<div class="description">
+								<p>Send a raw message to the current IRC network.</p>
+								<p>Aliases: <code>/quote</code>, <code>/send</code></p>
+							</div>
+						</div>
+
+						<div class="help-item">
+							<div class="subject">
+								<code>/slap nick</code>
+							</div>
+							<div class="description">
+								<p>Slap someone in the current channel with a trout!</p>
+							</div>
+						</div>
+
+						<div class="help-item">
+							<div class="subject">
+								<code>/topic newtopic</code>
+							</div>
+							<div class="description">
+								<p>Set the topic in the current channel.</p>
+							</div>
+						</div>
+
+						<div class="help-item">
+							<div class="subject">
+								<code>/voice nick [...nick]</code>
+							</div>
+							<div class="description">
+								<p>
+									Give voice (<code>+v</code>) to one or several users in the
+									current channel.
 								</p>
 							</div>
 						</div>
+
+						<div class="help-item">
+							<div class="subject">
+								<code>/whois nick</code>
+							</div>
+							<div class="description">
+								<p>
+									Retrieve information about the given user on the current
+									network.
+								</p>
+							</div>
+						</div>
+
+						<h2>Keyboard Shortcuts</h2>
```

Comment:
Could we stick the keyboard shortcuts at the top? It feels like that being a shorter section would make more sense to be at the top. Also, it's more likely to be the thing that people are initially wanting to look at.